cms/templates/container.html

```diff
@@ -8,27 +8,31 @@
 %>
 </%def>
 <%!
+from django.utils.translation import ugettext as _
+
 from contentstore.views.helpers import xblock_studio_url, xblock_type_display_name
 from openedx.core.djangolib.js_utils import (
     dump_js_escaped_json, js_escaped_string
 )
-from openedx.core.djangolib.markup import HTML, ugettext as _
+from openedx.core.djangolib.markup import escape, HTML
 %>
-<%block name="title">${xblock.display_name_with_default_escaped} ${xblock_type_display_name(xblock) | h}</%block>
+<%page expression_filter="h"/>
```

Comment:
Let's move this very near the top of the file, since it is a very important thing about the entire file.